recipe/build.sh

```diff
@@ -1,23 +1,6 @@
 #!/usr/bin/env bash
-
-cd "${SRC_DIR}/.."
-mkdir "src"
-mkdir "src/github.com"
-mkdir "src/github.com/opencontainers"
-mkdir "src/github.com/tianon"
-mv "${SRC_DIR}" "src/github.com/tianon/gosu"
-
-export RUNC_VERSION="0.1.0"
-curl -L https://github.com/opencontainers/runc/archive/v${RUNC_VERSION}.tar.gz | tar -xzC "src/github.com/opencontainers"
-mv "src/github.com/opencontainers/runc-${RUNC_VERSION}" "src/github.com/opencontainers/runc"
-
 export CGO_ENABLED=0
```

Comment:
Is this still needed? Meaning I know it is needed for this recipe, but not if it is still needed with the `go` compiler.

Reply:
I think this is not needed here. I am removing to see if the code still builds.

Here is the best description I've found about the CGO_ENABLED flag.
https://go-review.googlesource.com/c/go/+/12603/2/src/cmd/cgo/doc.go

Reply:
That review makes it sound like we still need to set `CGO_ENABLED=0` if we expect there to be no cgo usage.

FWIW this comes from [here]( https://github.com/tianon/gosu/blob/d2937f478b317e55a77e0e0ed4f79a333a6f5735/Dockerfile#L14-L15 ). I think they are mainly interested in building portable binaries. Though am not clear how they are not still in some way attached to the C runtime library. So this may be less necessary for us.

That said, I think we can table this for now. Was just curious since you know more about Go than I whether this made sense for us. We can save it for another discussion though.